src/transport/FabricTable.h

```diff
@@ -312,43 +310,42 @@ class ConstAdminIterator
     }
 };
 
-class DLL_EXPORT AdminPairingTable
+class DLL_EXPORT FabricTable
 {
 public:
-    CHIP_ERROR Store(AdminId id);
-    CHIP_ERROR LoadFromStorage(AdminId id);
-    CHIP_ERROR Delete(AdminId id);
+    CHIP_ERROR Store(FabricIndex id);
+    CHIP_ERROR LoadFromStorage(FabricIndex id);
+    CHIP_ERROR Delete(FabricIndex id);
```

Comment:
Just a note, but it's pretty important: AdminId is a zero-based index today, and deeply considered as such, while it is an ID "space" for spec compliance. Suggest the next change in the list altogether is to account for that difference.